portfolio/src/main/webapp/newScript.js

```diff
@@ -0,0 +1,11 @@
+/**Fethces the response from the server and adds it to the page */
```

Comment:
nit: use a space after /** and fetches is misspelled.

Reply:
Okay